video/out/mac/common.swift

```diff
@@ -395,7 +406,58 @@ class Common: NSObject {
     }
 
     func updateICCProfile() {
-        log.sendWarning("updateICCProfile not implemented")
+        guard let windowColorSpace = window?.screen?.colorSpace else {
+            log.sendWarning("Couldn't update ICC Profile, no color space available")
+            return 
+        }
+
+        colorSpace = windowColorSpace
+
+        if #available(macOS 10.11, *) {
+
+            // Experimental #HDR on #macOS        
+            var name = "" as CFString
+
+// HDR is supported fully only on macOS 10.15 and higher
+// some of the HDR color spaces are available on 10.14 though
+#if HAVE_MACOS_10_15_FEATURES
```

Comment:
that won't work. you use lower macOS features within this. like i said you should do it similar to https://github.com/mpv-player/mpv/blob/master/video/out/mac/title_bar.swift#L215-L254